salt/states/junos.py

```diff
@@ -33,7 +33,7 @@ def wrapper(*args, **kwargs):
 
 
 @resultdecorator
-def rpc(name, dest=None, format="xml", args=None, **kwargs):
+def rpc(name, rpc=None, dest=None, format="xml", args=None, **kwargs):
```

Comment:
Best to add parameters to the end of an existing definition, that way it stays backwards compatible

Reply:
reverted via https://github.com/saltstack/salt/pull/57281/commits/682cb0a08baa14e1fa4684163c5588cbda672d25#diff-442286d51776ddefc791bc7f15e42fc2R36